homeassistant/components/drop_connect/coordinator.py

```diff
@@ -42,3 +42,14 @@ async def set_bypass(self, value: int):
             self.config_entry.data[CONF_COMMAND_TOPIC],
             payload,
         )
+
+    async def set_protect_mode(self, value: str):
```

Comment:
```suggestion
    async def set_protect_mode(self, value: str) -> None:
```
(Can you also do this at the others?)

Reply:
Added in 323dffb78d9a05f17dd971b112e9da7f4e18ae07.